src/main/java/eu/ai4work/sws/model/VariableExplanation.java

```diff
@@ -0,0 +1,13 @@
+package eu.ai4work.sws.model;
+
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class VariableExplanation {
+    private double value;
+    private Map<String, Double> terms;
```

Comment:
I was wondering how to make the meaning more clear to users by using another word than "terms". The message to the user is that based on the input `value` the fuzzy engine calculated the "degree of true-ness of terms". Maybe one of the following would make sense?
```suggestion
    private Map<String, Double> membershipValues;
```
or 
```suggestion
    private Map<String, Double> membershipDegrees;
```
or
```suggestion
    private Map<String, Double> truthValues;
```